src/scripts/main.js

```diff
@@ -354,7 +354,31 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('table.dashboard');
 
-// write your code here
+function createRowInDashboard({ name: personName, sex, born, died }) {
+  const tableRow = document.createElement('tr');
+  const rowData = [
+    personName,
+    sex,
+    born,
+    died,
+    died - born,
+    Math.ceil(died / 100),
```

Comment:
The calculation `Math.ceil(died / 100)` might not correctly determine the century for years that are exactly a multiple of 100. Consider using `Math.ceil((died - 1) / 100) + 1` to handle such cases correctly.